redbot/cogs/mod/settings.py

```diff
@@ -365,3 +367,15 @@ async def defaultdays(self, ctx: commands.Context, days: int = 0):
                 days=days
             )
         )
+
+    @modset.command()
+    @commands.guild_only()
+    async def defaultduration(self, ctx: commands.Context, days: int = 0):
+        """Set the default number of days to be used when a user is tempbanned."""
+        guild = ctx.guild
+        if not (0 <= days):
```

Comment:
tempban only allow days to be between 1 or 7, it miss a check if the days parameter is longer than 7 days.

Reply:
Where did you see that? From what I can see only ``days`` needs to be within 0-7 but not ``duration``.

I feel like I have definitely tempbanned for a year before.

Reply:
It is line 464 in kickban.py (Using default branch).

Reply:
Yes, that is ``days`` not ``duration``